app/navigation/navigator.py

```diff
@@ -20,8 +26,23 @@ def _valid_destination(self, destination):
         :returns: True if valid, False otherwise
 
         """
-        # TODO Addressable elements for navigation within a questionnaire need to be decided
-        return True
+        if destination == 'introduction':
+            if self._schema.introduction is not None:
+                return True
+            else:
+                return False
+
```

Comment:
I'm not sure how else to do this... 'introduction' is only a valid destination if the schema defines one.  If there's no introduction defined, then it's not a valid destination within the schema